_data/locales/en.yml

```
@@ -391,3 +391,7 @@ Tumbleweed_features:
     name: Stands on Firm Ground
     description: |
       Tumbleweed builds on decades of usage, testing and debugging by hundreds of power-users, developers, system administrators and demanding doers that cannot afford to jeopardize their workflow. Tumbleweed's solidity is embodied in many core packages whose DNA stems from the venerable SUSE Linux Enterprise Server.
+  gaming:
+    name: Gaming
+    description: |
+     Tumbleweed follows a rolling release model, providing users with the latest software updates and features, including gaming-related packages. openSUSE Tumbleweed is designed to maintain stability, ensuring a reliable gaming experience. In short, you can expect epic gaming performance with stability.
```

We already explained stability and and latest packages being available above, let's instead talk about why those things matter specifically for gaming
```suggestion
     Tumbleweed provides users with the latest gaming-related software. With the Linux support for gaming improving rapidly, that's an important aspect for smooth, performant and problem free experience in your favorite games.
```

---

okay, I'll write more specific to gaming

---

should I push your suggested text or should I add a few more lines? @hellcp

---

I don't know, I'm fine with mine, but feel free to expand it